sonar-text-dotnet/pipeline_scripts/extract_project_version.ps1

```diff
@@ -0,0 +1,11 @@
+$versionFilePath = "$env:CIRRUS_WORKING_DIR\sonar-text-dotnet\Directory.Build.props"
```

Comment:
This script is more or less the same with [this one in the azure pipeline](https://github.com/SonarSource/sonar-secrets/blob/master/sonar-secrets-dotnet/azure-pipelines-dotnet.yml#L51).

I could not pull the version from the `csproj` (the `Version` propertyGroup does not exist there and when I pull another property group (for example `AssemblyVersion`) the env variables are not properly transformed to their values).


I pulled the version from the Directory.Build.props which is the file used to set the version in the first place ([see example of version bump in the old repo](https://github.com/SonarSource/sonar-secrets/commit/c417983fb496c1678f60a0d1740bddb71adbf59e)).